### PR TITLE
chore: clean:all

### PR DESCRIPTION
**Pull request by HavardNot**
cleans all packages and apps respective node_modules and dist/ios folder

### CODE REVIEW

**Review by tormodAase — DISMISSED** [review]
Lookin good 🔥

File: .changeset/unlucky-crabs-build.md
Comment (HaakonSvane) [inline .changeset/unlucky-crabs-build.md:1]: I don't think we need to create changesets for these kinds of changes. The changesets are the changelog-items that end up on users release notes for the packages. This change feels to me more like a developer tool for the frontend tigers working in the monorepo. Should we remove this?

File: package.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
         "dev:all": "turbo run dev",
         "lint:all": "turbo run lint --continue --",
         "format:all": "prettier --write .",
-        "publish:all": "turbo run build && changeset version && changeset publish"
+        "publish:all": "turbo run build && changeset version && changeset publish",
+        "clean:all": "turbo run clean"
```
Comment (HaakonSvane) [inline package.json:36]: Should this also delete `node_modules` in the root directory?

File: turbo.json [hunk 2]
```diff
@@ -16,6 +16,7 @@
         "generate": {
             "cache": false
         },
-        "test": {}
+        "test": {},
+        "clean": {}
```
Comment (HaakonSvane) [inline turbo.json:20]: Turbo's caching system seem to sometimes not want to delete `node_modules` after being ran once. I created a `node_modules` folder in one of the packages and ran the script again, but turbo cached it all and did not want to delete it. We should disable caching by adding a 
```json
"cache": false
```
field to this job

**Review by HaakonSvane — DISMISSED** [review]
Great job frontend dancer 👯 
It seems the maintenance api wrapper package was left out. Perhaps we should add the clean script to that as well

File: .changeset/unlucky-crabs-build.md
Comment (HavardNot) [reply .changeset/unlucky-crabs-build.md:1]: removed

File: package.json (hunk 1 above)
Comment (HavardNot) [reply package.json]: done

File: turbo.json (hunk 2 above)
Comment (HavardNot) [reply turbo.json]: done

**Review by HaakonSvane — APPROVED** [review]
👍

**Pull request merged**